### PR TITLE
feat(npm): package version suggestions

### PR DESCRIPTION
**Pull request by fedeci**
Based on #525, closes #525

### CODE REVIEW

File: src/npm.ts [hunk 1]
```diff
@@ -10,26 +10,77 @@ function uninstallSubcommand(named: string | string[]): Fig.Subcommand {
...
+    if (oldToken.startsWith("@")) {
+      return !(atsInStr(oldToken) > 1 && atsInStr(newToken) > 1);
+    }
+
+    // If the package name doesn't start with '@', then trigger when
+    // we see the first '@' so we can generate version suggestions
+    return !(oldToken.includes("@") && newToken.includes("@"));
   },
-  postProcess: function (out) {
+  getQueryTerm: "@",
```
Comment (fedeci) [inline src/npm.ts:30]:
```suggestion
  getQueryTerm: "@",
  cache: {
     ttl: 1000 * 60 * 60 * 24 * 2, // 2 days
   },
```

**Comment by fedeci** [comment]
There are still other improvements that could be made for this generator but I will leave them for a separate PR in the future.

**Review by a7med-mahmoud — APPROVED** [review]
Looks good to me

**Review by mschrage — APPROVED** [review]
Great work @fedeci

**Pull request merged**